src/modules/shared/Filtration/Filtration.module.scss

```diff
@@ -1,16 +1,33 @@
 @import '../../../styles/utils/variables/colors';
+@import '../../../styles/utils/mixins/mixin-media';
 
 .filtration {
   display: flex;
+  gap: 16px;
```

Comment:
I hopes no one would notice, because for me this version of design looks kind of ugly) But if we want to be more precise, sure, that the soluton.

Reply:
filtration fall out os container on mobile vertion you can check it i had to fix it and i made it close to our loyat